velox/functions/sparksql/aggregates/tests/CollectSetAggregateTest.cpp

```diff
@@ -88,6 +88,47 @@ TEST_F(CollectSetAggregateTest, global) {
       {data}, {}, {"collect_set(c0)"}, {"spark_array_sort(a0)"}, {expected});
 }
 
+TEST_F(CollectSetAggregateTest, noInputRow) {
+  // Empty input.
+  auto data = makeRowVector({makeFlatVector<int32_t>({})});
+  auto expected = makeRowVector({
+      makeArrayVectorFromJson<int32_t>({"[]"}),
+  });
+  testAggregations({data}, {}, {"collect_set(c0)"}, {}, {expected});
```

Comment:
nit: perhaps, test a case when input is not empty, but mask is false for all rows